src/components/data_submission/consent_group/EditConsentGroup.js

```diff
@@ -429,45 +429,52 @@ export const EditConsentGroup = (props) => {
           onChange({ key, value: value.dacId });
         }
       }),
+    ]),
 
-      // location
-      h(FormField, {
-        id: idx + '_dataLocation',
-        name: 'dataLocation',
-        type: FormFieldTypes.SELECT,
-        isMulti: true,
+    // location
+
+    div({style:{ display: 'flex', flexDirection:'row', justifyContent: 'space-between', marginBottom: '-5%' }}, [
+      h(FormFieldTitle, {
+        required: true,
         title: 'Data Location',
         description: 'Please provide the location of your data resource for this consent group',
-        exclusiveValues: ['Not Determined'],
-        selectOptions: [
-          'AnVIL Workspace',
-          'Terra Workspace',
-          'TDR Location',
-          'Not Determined',
-        ],
-        defaultValue: consentGroup.dataLocation,
-        placeholder: 'Data Location(s)',
-        validators: [FormValidators.REQUIRED],
-        onChange,
-        validation: validation.dataLocation,
-        onValidationChange,
-      }),
-
-      h(FormField, {
-        id: idx + '_url',
-        name: 'url',
-        title: 'Data URL',
-        validators: [FormValidators.REQUIRED, FormValidators.URL],
-        placeholder: 'Free text field for entering URL of data',
-        defaultValue: consentGroup.url,
-        onChange,
-        validation: validation.url,
-        onValidationChange,
       }),
     ]),
 
     h(FormTable, {
```

Comment:
quick nit: since the user doesn't have to add multiple rows, it would probably be easier to just do this in a div. something like:

```
div({ className: "flex flex-row" }, [ h(FormField, ...), h(FormField, ...) ]
```

Then, the ids wouldn't have to change for the tests and we'd have some more control over formatting